modules/API/API.js

```diff
@@ -428,6 +429,17 @@ function initCommands() {
                 logger.error('Failed sending endpoint text message', err);
             }
         },
+        'overwrite-name': (id, name) => {
+            logger.debug('Overwrite name command received');
+            let breakoutRoom = false, identifier = id;
+
+            if (id.indexOf('@') !== -1) {
```

Comment:
Can we hide this and take care of it inside `overwriteParticipantName` ? I think it makes for a cleaner API.